requirements.txt

```diff
@@ -8,3 +8,5 @@ ipywidgets
 sentence-transformers
 pylint
 streamlit
+pydantic
```

Comment:
It would be better to include guard logic so that we don't require these to install `prompttools` if the user is experimenting with other LLMs.

See LlamaCpp for example: https://github.com/hegelai/prompttools/blob/main/prompttools/experiment/experiments/llama_cpp_experiment.py#L12-L15

Reply:
Reverted